src/test/java/it/pagopa/transactions/services/v1/CircuitBreakerTest.java

```diff
@@ -54,9 +64,315 @@ class CircuitBreakerTest {
     @MockBean
     private TransactionActivateHandler transactionActivateHandlerV1;
 
+    @MockBean
+    private TransactionsViewRepository transactionsViewRepository;
+    @MockBean
+    private TransactionsUtils transactionsUtils;
+
     @Autowired
     private CircuitBreakerRegistry circuitBreakerRegistry;
 
+    @Autowired
+    private RetryRegistry retryRegistry;
+
+    private static final JsonNode resilience4jConfiguration;
+
+    private static final Map<String, Exception> exceptionMapper = Map.of(
+            "it.pagopa.transactions.exceptions.UnsatisfiablePspRequestException",
+            new UnsatisfiablePspRequestException(
+                    new PaymentToken(""),
+                    RequestAuthorizationRequestDto.LanguageEnum.IT,
+                    0
+            ),
+            "it.pagopa.transactions.exceptions.PaymentNoticeAllCCPMismatchException",
+            new PaymentNoticeAllCCPMismatchException("rptId", true, true),
+            "it.pagopa.transactions.exceptions.TransactionNotFoundException",
+            new TransactionNotFoundException(""),
+            "it.pagopa.transactions.exceptions.AlreadyProcessedException",
+            new AlreadyProcessedException(new TransactionId(TransactionTestUtils.TRANSACTION_ID)),
+            "it.pagopa.transactions.exceptions.NotImplementedException",
+            new NotImplementedException(""),
+            "it.pagopa.transactions.exceptions.InvalidRequestException",
+            new InvalidRequestException(""),
+            "it.pagopa.transactions.exceptions.TransactionAmountMismatchException",
+            new TransactionAmountMismatchException(10, 11),
+            "it.pagopa.transactions.exceptions.NodoErrorException",
+            new NodoErrorException(new CtFaultBean()),
+            "it.pagopa.transactions.exceptions.InvalidNodoResponseException",
+            new InvalidNodoResponseException("")
+    );
```

Comment:
I think this could be simplified to something like:

```suggestion
    private static final Map<String, Exception> exceptionMapper = Stream.of(new ..., new ..., new ...).collect(Collectors.toMap(e -> e.getClass().getName(), x -> x));
```

Reply:
Fixed with https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/370/commits/84942ffc696f5d9a21bdc6b7caeedff834b64d3e